loopchain/configure_default.py

```diff
@@ -419,6 +417,15 @@ def is_support_node_function(cls, node_function, node_type):
 CONF_PATH_ICONRPCSERVER_DEV = os.path.join(LOOPCHAIN_ROOT_PATH, 'conf/develop/iconrpcserver_conf.json')
 CONF_PATH_ICONRPCSERVER_TESTNET = os.path.join(LOOPCHAIN_ROOT_PATH, 'conf/testnet/iconrpcserver_conf.json')
 CONF_PATH_ICONRPCSERVER_MAINNET = os.path.join(LOOPCHAIN_ROOT_PATH, 'conf/mainnet/iconrpcserver_conf.json')
+ICON_VERSIONS = {
+    'loopchain': '0.0.0',
+    'iconservice': '0.0.0',
+    'iconrpcserver': '0.0.0',
```

Comment:
How about add iconrpcserver version info at `iconrpcserver.server.peer_service_stub.PeerServiceStub.get_status()`?

`/status/peer` API always passes `iconrpcserver` in and out, and version info of `iconrcpserver` can be added at its process time. I think it ensures get information of iconrpcserver version, even if it runs separate environment from `loopchain`.

Reply:
For those cases that are in the separate virtual environment, this could be solved by changing the default version value in the configuration.